features/step_definitions/assertion_steps.rb

```diff
@@ -5,3 +5,7 @@
 Then('I should not see {string}') do |content|
     expect(page).not_to have_content content
 end
+
+Then('show me the page') do
+    save_and_open_page
+end
```

Comment:
If you are done with debugging, remove this

Reply:
done